bucket/msxml4.json

```
@@ -0,0 +1,17 @@
+{
+    "##": "Renaming .msi to .msi_ so that the MSI package will not be automatically extracted by Scoop.",
+    "version": "4.30.2100.0",
+    "description": "Microsoft XML Core Services (MSXML) runtime library",
+    "homepage": "http://msdn.microsoft.com/en-us/library/ms763742.aspx",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://www.microsoft.com/en-us/legal/intellectualproperty/copyright/default.aspx"
+    },
+    "url": "https://download.microsoft.com/download/A/2/D/A2D8587D-0027-4217-9DAD-38AFDB0A177E/msxml.msi#/setup.msi_",
+    "hash": "47c2ae679c37815da9267c81fc3777de900ad2551c11c19c2840938b346d70bb",
+    "post_install": [
+        "if (-not (is_admin)) { error 'This package requires admin privileges to install'; Exit(1) }",
```

Never use exit or abort in manifests

```suggestion
        "if (-not (is_admin)) { error 'This package requires admin privileges to install'; break }",
```

Later all breaks will be refactored to throw after `abort` is fully deprecated